### PR TITLE
done intro

### PR DESCRIPTION
**Pull request by Ppllupold**

### CODE REVIEW

**Comment by Ppllupold** [comment]
image

**Comment by Ppllupold** [comment]
image
image
image

**Comment by Ppllupold** [comment]
image
image
image

**Comment by Ppllupold** [comment]
sorry for this mess, i screwed up with migrations while trying to squash squashed migrations, so i can`t commit in this pull request anymore, i don`t know what should i do. I have migration conflict. Should i just create another fork and rewrite everything again?

**Comment by Ppllupold** [comment]
image

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+.env
+
+__pycache__/
+*.py[cod]
+
+.idea/
+.vscode/
+
+
```
Comment (viktoria-rybenchuk) [inline .gitignore:14]: remove empty lines

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**